### ISSUE TITLE
custom dataset:should I add the non-object images to the training data?

### ISSUE THREAD

**Issue by hazy-wu** [qqwweee/keras-yolo3#663]
I wanna create a custom dataste and train the network, so I set all images, including the non-object images as datas, but finally I found it has bad performance while I remove the background-only images and it has a better performance. So I wanna know whether this network supports to input the background-only images, aiming to improve the performance? It's comfusing. If you can reply me, I will appreciate you sincerely.

**Comment by nrgrady** [comment]
I would also like to know how to do this in this port of YOLOv3. 
Using negative images without labels seems to be a standard and straight forward process in the new YOLOv4, but I've had problems getting it to work on this repo.
